portfolio/src/main/java/com/google/sps/servlets/AnimeDataServlet.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/anime-data")
+public class AnimeDataServlet extends HttpServlet {
+  
```

Comment:
nit: Can you delete this line?

Reply:
which line are you referring to?

Reply:
Oh.. I missed that. I meant the blank line after the class definition. One of the important Google style philosophy is to put more code in one screen (avoid unnecessary spacing), and removing this blank line is one example of that. You do want to use blank lines if it increases the readability (e.g., to add semantical separation).

Reply:
Ahh I see, I'll just remove that line then. I'll be gone with my next PR that I'll send shortly